.circleci/config.yml

```diff
@@ -385,6 +385,116 @@ jobs:
       - store_artifacts:
           path: packages/python/plotly/dist
 
+
+  build-doc:
+    resource_class: xlarge
+    docker:
+      # specify the version you desire here
+      # use `-browsers` prefix for selenium tests, e.g. `3.6.1-browsers`
+      - image: circleci/python:3.6.8-node
+
+
+        working_directory: ~/project
+
+    steps:
+      - add_ssh_keys:
+          fingerprints:
+            - "d9:b0:da:5c:3e:8c:b6:b6:d5:b6:35:70:ce:fe:f5:b1"
+
+      - checkout
+
+      # Download and cache dependencies
+      - restore_cache:
+          keys:
+            - v1-dependencies-{{ checksum "doc/requirements.txt" }}
+            # fallback to using the latest cache if no exact match is found
+            - v1-dependencies-
+
+      - run:
+          name: install dependencies
+          command: |
+            cd doc
+            python3 -m venv venv
+            . venv/bin/activate
+            npm install electron@1.8.4
```

Comment:
I'm curious about why you needed a different set of things here than I did in the other repo...?

Was it just `npm install orca` that was missing?

Reply:
It's the same instructions I think, I just put the installation of electron and orca on two separate lines for the sake of debugging, I had trouble with finding the path where orca had been installed.